lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,72 @@
+require 'json_downloader'
+
+class LocalAuthoritiesImporter
+  include JsonDownloader
+
+  DISTRICT = 'district'
+  COUNTY = 'county'
+  UNITARY = 'unitary'
+
+  LOCAL_AUTHORITY_MAPPING = {
+    "COI" => UNITARY,
+    "CTY" => COUNTY,
+    "DIS" => DISTRICT,
+    "LBO" => UNITARY,
+    "LGD" => UNITARY,
+    "MTD" => UNITARY,
+    "UTA" => UNITARY,
+  }
+
+  def self.import_from_mapit
+    new.authorities_from_mapit
+  end
+
+  def authorities_from_mapit
+    url = "#{MAPIT_BASE_URL}areas/#{local_authority_types}"
+    mapit_las = get_mapit_authorities(url)
```

Comment:
We'd usually use gds-api-adapters for internal API calls - there's a [method there which makes this call](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/mapit.rb#L11-L13) (although its name suggests that that endpoint only accepts a single type).